openfisca_us/parameters/gov/states/in/tax/income/taxes/county/rates.yaml

```diff
@@ -0,0 +1,197 @@
+description: IN county tax rates.
+metadata:
+  period: year
+  unit: /1
+  name: in_county_tax_rates
+  label: IN county tax rates
+  reference:
+  - title: IC 6-3.6 ARTICLE 3.6. LOCAL INCOME TAXES
+    href: http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3.6
+
+  breakdown:
+    - county_enum
```

Comment:
Could you remove the breakdown? The state-only performance tricks mean we don't have to use it, and it'd increase performance by not having to add a zero rate for every non-IN county.